code/game/world.dm

```diff
@@ -339,8 +342,12 @@ GLOBAL_VAR_INIT(world_topic_spam_protect_time, world.timeofday)
 	return
 	#endif
 
+	var/secs_before_auto_reconnect = 10
+	if(GLOB.pending_server_update)
+		secs_before_auto_reconnect = 60
```

Comment:
Could be a made a config option. Would also allow the automatic changing of the waiting time depending on how long it took the times before hand.
All very overkill for what it is though

Reply:
@farie82 it takes 60 seconds for an update by default, if you include the restart overhead too. A config option is not only necessary for this, it wouldnt be useful since the backend doesnt modify the config in any way.

Reply:
That's on our hardware on our network. It can take longer I'd guess on other setups and seeing this is an open source repo. But like I said it's fully optional

Reply:
@farie82 This entire feature is also custom-specific to our backend tooling. I only know of one "real" downstream of us, and they host on TGS, which doesnt have reboot delays.